data/35e/wizards_of_the_coast/supplement/book_of_vile_darkness/bv_classes.lst

```diff
@@ -359,6 +343,7 @@ CLASS:Thrall of Orcus	STARTSKILLPTS:2	CSKILL:Bluff|Concentration|TYPE=Craft|Hide
 ###Block
 1	ABILITY:Thrall of Orcus Class Feature|AUTOMATIC|Thrall of Orcus ~ Weapon and Armor Proficiency
 1	ABILITY:Thrall of Orcus Class Feature|AUTOMATIC|Thrall of Orcus ~ Bonus Feat or Spells
+1:REPEATLEVEL:3	BONUS:ABILITYPOOL|Thrall of Orcus Choice|1
```

Comment:
Out of standard convention. BONUS should be in the class feature with the formula based off the class level.